<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SBND Purity Monitor DAQ</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="PlotWidget" name="_plot" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>460</y>
      <width>921</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="_logs_btn">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>700</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Logs</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1001</width>
      <height>321</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="_vertical_layout"/>
   </widget>
   <widget class="QPushButton" name="_settings_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>700</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionShow_Logs"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionShow_Logs">
   <property name="text">
    <string>Show Logs</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
